ArduCopter/mode_auto.cpp

```diff
@@ -1991,12 +1991,12 @@ bool ModeAuto::verify_payload_place()
     case PayloadPlaceStateType_Ascending_Start: {
         Location target_loc(inertial_nav.get_position_neu_cm(), Location::AltFrame::ABOVE_ORIGIN);
         target_loc.alt = nav_payload_place.descend_start_altitude;
-        wp_start(target_loc);
+        auto_takeoff_start(target_loc.alt, target_loc.get_alt_frame() == Location::AltFrame::ABOVE_TERRAIN);
         nav_payload_place.state = PayloadPlaceStateType_Ascending;
         }
         FALLTHROUGH;
     case PayloadPlaceStateType_Ascending:
-        if (!copter.wp_nav->reached_wp_destination()) {
+        if (!verify_takeoff()) {
```

Comment:
I think we should probably use, "auto_takeoff_complete" instead of "verify_takeoff" because re-using another command's verify is something we haven't done before.  Future developers wouldn't expect this.